lib/win32/taskscheduler/helper.rb

```diff
@@ -12,11 +12,11 @@ module Helper
       ffi_lib :kernel32, :advapi32
 
       attach_function :FormatMessage, :FormatMessageA,
-      [:ulong, :pointer, :ulong, :ulong, :pointer, :ulong, :pointer], :ulong
+                      %i{ulong pointer ulong ulong pointer ulong pointer}, :ulong
```

Comment:
It looks like you actually ran rubocop against the codebase. That's fine at the moment, but in the future we use chefstyle for all our Chef projects.

Reply:
Yes, since chefstyle ignores many patterns, so I usually run rubocop first and then chefstyle again. Will avoid running rubocop if its not required.